### ISSUE TITLE
Allow to ask for retries on JES error 13

### ISSUE THREAD

**Issue by Horneth** [broadinstitute/cromwell#1849]
Google asked us to treat both 13 and 14 identically wrt to preemption, as they might send back one or the other.
However, 13 can also happen for not preempted reasons (e.g if the VM is shutdown manually, or some other transient shutdown).
Design a way so the user can ask for 13s to be retried even for non-preemptible VMs

**Comment by cjllanwarne** [comment]
Might this eventually expand to be a more general purpose "allow certain error codes to be retried"?

**Comment by Horneth** [comment]
@cjllanwarne I think yes it definitely could.

**Comment by kcibul** [comment]
+100!

**Comment by geoffjentry** [comment]
Oops I just opened #1888 which is effectively a dup of this. I'll just add some notes here instead.

We discussed in person that the key stakeholders wanted this to always happen a couple of times and that we would not expose this as an option for now. 

IMO I would prefer to see this sort of JES-backend-specific retrying being handled as part of the backend's responsibility. I haven't looked closely at the code to assess how much that desire makes sense
